### ISSUE TITLE
One of the returned values {'metrics', 'm_outputs'} has a `grad_fn`. We will detach it automatically but this behaviour will change in v1.6. Please detach it manually: `return {'loss': ..., 'something': something.detach()}`

### ISSUE THREAD

**Issue by thibo73800** [Visual-Behavior/aloception#16]
Using DETR we get the following warning

```
One of the returned values {'metrics', 'm_outputs'} has a `grad_fn`. We will detach it automatically but this behaviour will change in v1.6. Please detach it manually: `return {'loss': ..., 'something': something.detach()}`
```
